samples/ProjectGrandmaster.Unity/Assets/AudioSystem.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+[RequireComponent(typeof(AudioSource))]
+public class AudioSystem : MonoBehaviour
```

Comment:
Consider naming this something better, as System implies singleton, and based on this we could drop it on many components?